### ISSUE TITLE
Try to selcted only possible or supported extension with input box.

### ISSUE THREAD

**Issue by ankitkanojia** [DominicTobias/react-image-crop#326]
Right now video and other irrelevant files are being selected, do prevention of those file selection.

**Comment by DominicTobias** [comment]
Hi I;m not sure what you are talking about - the demo here? https://codesandbox.io/s/72py4jlll6

It works ok for me in this regard, please add more detail

**Comment by ankitkanojia** [comment]
Yeah, it seems to work perfectly as expected. But when you select any other format below are the steps

1. select any image and crop it
2. now select any other video or unsupported files both files will be there.

So, think just broke the functionality when select unsupported media after supported media.

Otherwise its work perfectly, It's awwsome.....

**Comment by ankitkanojia** [comment]
For suggestion purpose:

You can restrict all unsupported media in HTML input type file issue will be resolve automatically.

**Comment by DominicTobias** [comment]
Ah ok well this is perhaps an issue with the demo, which is just intended as a simple demo to show the library not as perfect behaviour in an app. Although after selecting something that isn't an image I saw the previous image that was selected and nothing changed except the file name of the selected item and I was able to select something else after so didn't seem like an issue to me. Anyway I've added an `accept` attribute to only select images now.
